stubs/jmespath/jmespath/exceptions.pyi

```diff
@@ -1,44 +1,41 @@
+from collections.abc import Sequence
 from typing import Any
 
 class JMESPathError(ValueError): ...
 
 class ParseError(JMESPathError):
-    lex_position: Any
-    token_value: Any
-    token_type: Any
-    msg: Any
-    expression: Any
-    def __init__(self, lex_position, token_value, token_type, msg=...) -> None: ...
+    lex_position: int
+    token_value: str
+    token_type: str
+    msg: str
+    expression: str | None
+    def __init__(self, lex_position: int, token_value: str, token_type: str, msg: str = ...) -> None: ...
 
 class IncompleteExpressionError(ParseError):
-    expression: Any
-    lex_position: Any
-    token_type: Any
-    token_value: Any
-    def set_expression(self, expression) -> None: ...
+    expression: str
+    def set_expression(self, expression: str) -> None: ...
 
 class LexerError(ParseError):
-    lexer_position: Any
-    lexer_value: Any
-    message: Any
-    expression: Any
-    def __init__(self, lexer_position, lexer_value, message, expression: Any | None = ...) -> None: ...
+    lexer_position: int
+    lexer_value: str
+    message: str
+    def __init__(self, lexer_position: int, lexer_value: str, message: str, expression: str | None = ...) -> None: ...
 
 class ArityError(ParseError):
-    expected_arity: Any
-    actual_arity: Any
-    function_name: Any
-    expression: Any
+    expected_arity: int
+    actual_arity: int
+    function_name: str
+    expression: None
```

Comment:
Feels hard to believe that this is _always_ `None`, otherwise there's not much point in having it as an attribute ;)

Maybe just don't override it here, and let it inherit the `str | None` annotation from the superclass?

```suggestion
```